CHANGELOG.rst

```diff
@@ -3811,7 +3811,6 @@ Bug Fixes
 .. _@philpep: https://github.com/philpep
 .. _@raquel-ucl: https://github.com/raquel-ucl
 .. _@axil: https://github.com/axil
-.. _@tgoodlet: https://github.com/tgoodlet
```

Comment:
fwiw, now @goodboy

Reply:
Heh, yeah sorry about any confusion on that one.
I finally got the username I always wanted a few months back.